### DISCUSSION TITLE
Script to send the content of currently selected text to Zipline

### DISCUSSION THREAD

**Discussion by asitemade4u** [diced/zipline#391] [Questions] [answered]
Hi,
First, let me thank you for this excellent and comprehensive application: thanks to the excellent documentation, I had no difficulty deploying it using docker and tweaking it to our needs.

I would now like to **use Zipline as a Pastebin**.
I have tried to adapt the scripts you provided but to no avail. I am using Manjaro/Arch but I guess a bash script would also be applicable to other flavors of Linux, at least in the debian perimeter.

Could you give me a framework for such a script? (and add it to the documentation as I am sure it would interest others).

Thank you in advance.
Stephen

**Comment by TacticalTechJay** [comment, marked as answer]
Perhaps this can work for your use case:

```bash
xsel -ob | curl -H "authorization: $TOKEN" -F file=@- -F type=text/plain $HOST/api/upload
```

`xsel -ob` will read what is in the clipboard and pipe it to curl. Not sure if I can get zipline to respond with extensions like `.txt` or `.js` though...
